modules/vm_workload_scanning.cft.yaml

```diff
@@ -0,0 +1,260 @@
+        Active: true
+      OperationPreferences:
+        MaxConcurrentPercentage: 100
+        FailureTolerancePercentage: 90
+        ConcurrencyMode: SOFT_FAILURE_TOLERANCE
+      Parameters:
+        - ParameterKey: NameSuffix
+          ParameterValue:
+            Ref: NameSuffix
+        - ParameterKey: TrustedIdentity
+          ParameterValue:
+            Ref: TrustedIdentity
+        - ParameterKey: ExternalID
+          ParameterValue:
+            Ref: ExternalID
+        - ParameterKey: LambdaScanningEnabled
+          ParameterValue:
+            Ref: LambdaScanningEnabled
+      StackInstancesGroup:
+        - DeploymentTargets:
+            OrganizationalUnitIds: !Ref OrganizationalUnitIDs
+          Regions:
+            - Ref: AWS::Region
+      TemplateBody: |
+        AWSTemplateFormatVersion: "2010-09-09"
+        Description: IAM Role for Agentless Workload Scanning
+        Parameters:
+          NameSuffix:
+            Type: String
+            Description: Suffix to append to the resource name identifiers
+            AllowedPattern: "[0-9a-z]+"
+            MaxLength: 8
+            MinLength: 4
+          TrustedIdentity:
+            Type: String
+            Description: Trusted identity
+          ExternalID:
+            Type: String
+            Description: external ID
+          LambdaScanningEnabled:
+            Type: String
+            Description: Enable Lambda function scanning
+            Default: 'false'
+        Conditions:
+          IsLambdaEnabled:
+            Fn::Equals:
+              - Ref: LambdaScanningEnabled
+              - 'true'
+        Resources:
+          ScanningRole:
+            Type: AWS::IAM::Role
+            Properties:
+              RoleName: !Sub sysdig-vm-workload-scanning-${NameSuffix}
+              AssumeRolePolicyDocument:
+                Version: "2012-10-17"
+                Statement:
+                - Effect: "Allow"
+                  Action: "sts:AssumeRole"
+                  Principal:
+                    AWS: !Ref TrustedIdentity
+                  Condition:
+                    StringEquals:
+                      sts:ExternalId: !Ref ExternalID
+            ECRPolicy:
+              Type: AWS::IAM::Policy
+              Properties:
+                PolicyName: !Sub sysdig-vm-workload-scanning-${NameSuffix}-ecr
+                Roles:
+                  - !Ref ScanningRole
+                PolicyDocument:
+                  Version: '2012-10-17'
+                  Statement:
+                    - Effect: Allow
+                      Action:
+                        - ecr:GetDownloadUrlForLayer
+                        - ecr:BatchGetImage
+                        - ecr:BatchCheckLayerAvailability
+                        - ecr:ListImages
+                        - ecr:GetAuthorizationToken
+                      Resource: '*' 
+            LambdaPolicy:
+              Type: AWS::IAM::Policy
+              Condition: IsOrganizationalAndLambdaEnabled
```

Comment:
@miguelpais we don't have the `IsOrganizationalAndLambdaEnabled` conditional defined anywhere anymore. so this should be `IsLambdaEnabled` right?

Reply:
thank you so much! Addressed!